src/AppInstallerRepositoryCore/Microsoft/Schema/1_1/Interface_1_1.cpp

```diff
@@ -29,25 +29,36 @@ namespace AppInstaller::Repository::Microsoft::Schema::V1_1
     {
         std::vector<Utility::NormalizedString> GetSystemReferenceStrings(
             const Manifest::Manifest& manifest,
-            std::function<const Utility::NormalizedString&(const Manifest::ManifestInstaller&)> func)
+            std::function<const Utility::NormalizedString&(const Manifest::ManifestInstaller&)> extractStringFromInstaller,
+            std::optional<std::function<const Utility::NormalizedString& (const Manifest::AppsAndFeaturesEntry&)>> extractStringFromAppsAndFeaturesEntry = {})

```

Comment:
A `std::function` can already be empty; no need for a `std::optional` wrapper.

Reply:
Thanks. Didn't know that